RFC/src/RFC-0341_AssetRegistration.md

```diff
@@ -1 +1,94 @@
+
+## Language
+
+The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED",
+"NOT RECOMMENDED", "MAY", and "OPTIONAL" in this document are to be interpreted as described in
+[BCP 14](https://tools.ietf.org/html/bcp14) (covering RFC2119 and RFC8174) when, and only when, they appear in all capitals, as
+shown here.
+
+## Disclaimer
+
+The purpose of this document and its content is for information purposes only and may be subject to change or update
+without notice.
+
+This document may include preliminary concepts that may or may not be in the process of being developed by the Tari
+community. The release of this document is intended solely for review and discussion by the community regarding the
+technological merits of the potential system outlined herein.
+
+## Goals
+
+This document will describe the process that an [Asset Issuer] will need to engage in to register a [digital asset] and commence its operation on the [digital asset network].
+
+## Related RFCs
+* [RFC-0311: Digital Asset templates](RFC-0311_AssetTemplates.md)
+* [RFC-0302: Validator Nodes](RFC-0302_ValidatorNodes.md)
+* [RFC-0304: Validator Node committee selection](RFC-0304_VNCommittees.md)
+* [RFC-0220: Asset Checkpoints](RFC-0220_AssetCheckpoints.md)
+
+## Description
+
+### Abstract
+This document will describe the process an [Asset Issuer] (AI) will go through in order to:
+- Register a [digital asset] (DA) on the [base layer],
+- assemble a [committee] of [Validator Node]s (VNs) and
+- commence operation of the (DA) on the [Digital Asset Network] (DAN).
+
+### Asset creation instruction
+The first step in registering and commencing the operation of an asset is for the AI to issue an asset creation transaction to the [base layer].
+This transaction will be time-locked for the length of the desired nomination period. This ensures that this transaction cannot be spent until the nomination period has elapsed so that it is present during the entire nomination process. The value of the transaction will be the `asset_creation_fee` described in [RFC-0311](RFC-0311_AssetTemplates.md). The AI will spend the transaction back to themselves but locking this fee up at this stage achieves 2 goals. Firstly, it makes it expensive to spam the network with asset creation transactions that a malicious AI does not intend to complete. Secondly, it proves to the VNs that participate in the nomination process that the AI does indeed have the funds required to commence operation of the asset once the committee has been selected. If the asset registration process fails, for example if there are not enough available VNs for the committee, then the AI can refund the fee to themselves after the time-lock expires.
+
+The transaction will contain the following extra meta-data to facilitate the registration process:
+
+1. The value of the transaction in clear text so that it can be verified by third parties.
```

Comment:
We need some maths here to illustrate how this will work, how it will be proven that the clear text value is the same value committed to in the Pedersen Commitment.

Reply:
We either need to provide X.G as well or we need some view key

Reply:
The next line describes providing the public key but I will add in some brief maths.

Reply:
Updated this to explain the verification process.